docs/installation.rst

```diff
@@ -103,3 +104,108 @@ you make a change.
 The ``[dev]`` (a so-called "extra") installs additional development related dependencies,
 for example to run the unit tests.
 
+
+
+
+Development Installation on Windows
```

Comment:
This is adds a lot of dev-specific documentation on general installation page, which might intimidate a reader that is not closely looking

It's maybe better to put this in docs/development.rst

Reply:
That makes sense indeed.  I moved it to docs/development.rst

I also updated that section content-wise.
It looks like with the current code a dev installs works find in a virtualenv on Windows.
But maybe it is prudent to keep the documented solutions around, until we are sure this never happens anymore.